### PR TITLE
Added new testimonial

### PR DESCRIPTION
**Pull request by Ashish8689**
- Added new testimonial
- Added customer name in the testimonial.

image

### CODE REVIEW

**Review by ShilpaVernekar — APPROVED** [review]
Thanks @Ashish8689  Looks good

**Review by ShilpaVernekar — COMMENTED** [review]
Just need the name to be aligned with the right border space
image

**Comment by chirag-madlani** [comment]
> Just need the name to be aligned with the right border space @Ashish8689 image

@ShilpaVernekar It won't look good if we justify-content.
@Ashish8689 Do share a screenshot so that @ShilpaVernekar get more context

File: css/styles.css [hunk 1]
```diff
@@ -21014,7 +21014,7 @@ a:hover {
 
 
 .testimonial-card{
-  min-height: 358px;
+  min-height: 378px;
```
Comment (chirag-madlani) [inline css/styles.css:21017]: Not sure why need to increase min-height it's not the proper solution that you are looking for.
Find a way to have consistent height across testimonials

File: index.html [hunk 2]
```diff
@@ -368,9 +368,9 @@ <h2 class="display-5 mb-3 text-dark">Trusted Across Industries</h2>
               </div>
               <div class="swiper mySwiper">
                 <div class="swiper-wrapper" id="testimonialContainer"></div>
-                  <!-- <div class="swiper-button-next"></div>
+                  <div class="swiper-button-next"></div>
                   <div class="swiper-button-prev"></div>
-                <div class="swiper-pagination"></div> -->
+                  <!-- <div class="swiper-pagination"></div> -->
```
Comment (chirag-madlani) [inline index.html:373]: Why it's commented?

**Comment by ShilpaVernekar** [comment]
@chirag-madlani - Not getting the text justified....
@Ashish8689 and I were on a call and he was displaying the various options...

Testimonial center aligned looks better, but name center aligned felt as if something's missing (maybe if we add job title, it would look balanced)..... but since we are not adding job title, was trying to get the person's name right aligned.... but without that right side margin getting encroached

**Comment by Ashish8689** [comment]
@chirag-madlani @ShilpaVernekar will try some other combinations and will let you know both.

**Comment by Ashish8689** [comment]
@chirag-madlani @ShilpaVernekar have made the changes and set the description to left align in the card.

File: index.html (hunk 2 above)
Comment (Ashish8689) [reply index.html]: @chirag-madlani Removed!

**Review by ShilpaVernekar — APPROVED** [review]
Looks good

**Comment by chirag-madlani** [comment]
@Ashish8689 It's not good that arrows are overlapping text can you update position of arrows?

**Comment by Ashish8689** [comment]
@chirag-madlani Updated the cursor you can view the ScreenShot in above description

**Pull request merged**